calico_node/Makefile

```diff
@@ -529,8 +540,8 @@ semaphore:
 	# Actually run the tests (refreshing the images as required), we only run a
 	# small subset of the tests for testing SSL support.  These tests are run
 	# using "latest" tagged images.
-	$(MAKE) RELEASE_STREAM=$(RELEASE_STREAM) $(NODE_CONTAINER_NAME) st
-	ST_TO_RUN=tests/st/policy $(MAKE) RELEASE_STREAM=$(RELEASE_STREAM) st-ssl
+	$(MAKE) RELEASE_STREAM=master $(NODE_CONTAINER_NAME) st
+	ST_TO_RUN=tests/st/policy $(MAKE) RELEASE_STREAM=master st-ssl
```

Comment:
There's a bit of a general problem in this `Makefile` about how `RELEASE_STREAM` gets propagated (or not) to sub-make invocations, and I suspect that it's this that is causing you trouble with this PR.  Specifically:

1. I'm not sure that adding `RELEASE_STREAM=master` in these two lines here has wide enough scope to affect all the steps that `RELEASE_STREAM` should affect.  For example it doesn't affect the preceding `$(MAKE) node-test-containerized` line, and it could be that that line has already built some relevant components with the wrong `RELEASE_STREAM`.  Personally I always run make with `RELEASE_STREAM=master` in the environment of the top-level make invocation, e.g. `RELEASE_STREAM=master make st`, and I think we should make semaphore do the same, instead of this change here.

2. When we have a make step that does `docker run ...  make ...`, again `RELEASE_STREAM` doesn't get propagated - resulting in scary version output with obviously wrong versions.  I think we should fix this with something like adding `-e RELEASE_STREAM=$(RELEASE_STREAM)` to those steps.

I suspect that (1) is your actual problem here.

I *think* that (2) is benign - because I don't think of the steps build components that affect the ST as a whole - but it's probably worth fixing anyway.

Reply:
These items may be true, however they don't affect the outcome of the STs. Also if these are issues, they're also issues in `master` since the Makefile is the same in the areas you called out.

Reply:
How about we make that a variable declared at the top next to `RELEASE_STREAM ?= v2.6` say `CI_ RELEASE_STREAM ?= master` so it can be better consumed in the makefile and if it changes there's only one (obvious) place we need to change. WDYT?

Reply:
Otherwise, the changes LGTM